docs/_includes/Types.md

```diff
@@ -3472,6 +3490,8 @@ If the compiler does specialize the run-time representation for
 `int32`, then these statements come at the expense of two,
 respectively three, run-time conversions.
 
+TO BE WRITTEN - change of int(x) to x as int
```

Comment:
The old syntax `int(x)` went away a long time ago. Let's not mention it at all in the reference manual.

I submitted https://github.com/dafny-lang/dafny/issues/799 to remove this syntax from the implementation.